blueflood-core/src/integration-test/java/com/rackspacecloud/blueflood/io/astyanax/ACassandraUtilsIO.java

```diff
@@ -12,7 +13,7 @@
 /**
  * Utility class to provide some test methods existing tests use.
  */
-public class AstyanaxCassandraUtilsIO implements CassandraUtilsIO {
+public class ACassandraUtilsIO implements CassandraUtilsIO {
 
```

Comment:
I am not getting/understanding this AXXXX naming scheme:  ACassandraUtilsIO, ALocatorIO, AEnumIO, AMetadataIO, etc...  What's the idea here?